cosmos/client.go

```diff
@@ -232,9 +244,20 @@ func (c *Client) sign(msg sdktypes.Msg) (tenderminttypes.Tx, error) {
 		txBuilder = txBuilder.WithGas(adjusted)
 	}
 
-	signedTx, err := txBuilder.BuildAndSignStdTx(msg, c.accName, c.accPassword)
+	// create StdSignMsg
+	stdSignMsg, err := txBuilder.BuildSignMsg(msgs)
 	if err != nil {
 		return nil, err
 	}
-	return txBuilder.Encode(signedTx)
+
+	// create StdTx
+	stdTx := authtypes.NewStdTx(stdSignMsg.Msgs, stdSignMsg.Fee, []authtypes.StdSignature{}, stdSignMsg.Memo)
```

Comment:
```suggestion
	stdTx := authtypes.NewStdTx(stdSignMsg.Msgs, stdSignMsg.Fee, nil, stdSignMsg.Memo)
```

if `false` is passed to SignStdTx method, then we don't need an empty sigs here.